qutebrowser/mainwindow/prompt.py

```diff
@@ -467,6 +467,27 @@ def prompt_yank(self, sel=False):
         utils.set_clipboard(question.url, sel)
         message.info("Yanked to {}: {}".format(target, question.url))
 
+    @cmdutils.register(
+        instance='prompt-container', scope='window',
+        modes=[usertypes.KeyMode.prompt])
+    def prompt_external_picker(self):
+        """Choose a location using a configured external picker."""
+        picker = config.val.fileselect.folder.command
+        if not picker:
+            log.webview.warning(
```

Comment:
```suggestion
            log.prompt.warning(
```

would probably be more appropriate - but I suppose this should actually be a

```suggestion
            raise cmdutils.CommandError(
```

because the command cannot continue, and this should be communicated to the user (not just logged to the terminal, which is often missing or ignored.